platform/linux-generic/pktio/loop.c

```diff
@@ -315,6 +317,11 @@ static int loopback_send(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 		 */
 		out_octets_tbl[i] = bytes;
 		nb_tx++;
+
+		if (tx_ts_enabled && tx_ts_idx == 0) {
+			if (odp_unlikely(packet_hdr(pkt_tbl[i])->p.flags.ts_set))
```

Comment:
Wouldn't it be better to first check the unlikely packet header flag and only then tx_ts_enabled and tx_ts_idx? If time stamping is not enabled, the check for tx_ts_enabled would change to a check of the ts_set flag, which is should not be significantly more costly (wouldn't the flags be hot in the cache already?), but when time stamping is enabled, we would still need only single comparison instead of three for the vast majority of packets.

Reply:
And the same comment for the other occurences of the similar code in other commits in performance relevant pktios.

Reply:
Based on testing reading the `pkt_tbl[i])->p.flags.ts_set` first is more costly compared to the current implementation.